### PR TITLE
py-torchaudio: Submission

### PR DESCRIPTION
**Pull request by essandess**
###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS x.y
Xcode x.y

###### Verification 
Have you

- [ ] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [ ] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [ ] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [ ] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [ ] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?

### CODE REVIEW

**Comment by cjones051073** [comment]
@essandess can you check the build failures ?

```../../torchaudio/csrc/sox/utils.h:4:10: fatal error: 'sox.h' file not found
#include <sox.h>
```

File: python/py-torchaudio/Portfile [hunk 1]
```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+github.setup        pytorch audio 0.8.1 v
+revision            0
+name                py-[strsed ${github.author} {s|^py||}]${github.project}
```
Comment (reneeotten) [inline python/py-torchaudio/Portfile:9]: please just set this directly to `py-torchaudio`; that's immediately clear for everyone and there is no need to be overly clever here

Comment (cjones051073) [reply python/py-torchaudio/Portfile:9]: Yeah, I also noticed this, and almost made the same comment in other ports that did the same, but in the end let it pass. Normally I would be the first to support removing code/variable duplication, but here yeah its probably better to just write it out in plain text.

Comment (essandess) [reply python/py-torchaudio/Portfile]: *sigh* one less BSD regex in the world.

File: python/py-torchaudio/Portfile [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+PortGroup           python 1.0
+
+github.setup        pytorch audio 0.8.1 v
+revision            0
+name                py-torchaudio
+
+categories-append   audio
+platforms           darwin
+supported_archs     noarch
+license             BSD-2
```
Comment (cjones051073) [inline python/py-torchaudio/Portfile:14]: Can you fix the lint error here regarding the license type, see above.

port lint --nitpick py38-torchaudio:
Error: Invalid license 'BSD-2': use BSD instead

**Review by cjones051073 — CHANGES_REQUESTED** [review]

File: python/py-torchaudio/Portfile (hunk 2 above)
Comment (essandess) [reply python/py-torchaudio/Portfile]: Thanks done. Also removed the `noarch` as this compiles against `libsox`.

**Review by cjones051073 — APPROVED** [review]

**Pull request merged**